src/libraries/Microsoft.PowerFx.Core/Errors/BaseError.cs

```diff
@@ -208,14 +208,16 @@ private void FormatInnerError(StringBuilder sb)
             sb.AppendLine();
             var innerError = InnerError as BaseError;
             Contracts.AssertValue(innerError);
-            innerError?.Format(sb);
+            innerError?.Format(sb, culture);
         }
 
         public override string ToString()
         {
-            var sb = new StringBuilder();
-            Format(sb);
-            return sb.ToString();
+            throw new Exception();
```

Comment:
>throw new Exception(); [](http://example.com/codeflow?start=12&length=22)

this may likely break something, especially on PA side.

Reply:
intended, I want to confirm if it's used or not (we don't)